Planet Descriptions updates by Improvise (#7747)
endless-sky/endless-sky: data/map.txt

planet Alix
	attributes ka'het uninhabited
	landscape land/canyon13

> 	description `Alix is a small moon of the gas giant Pathera, lacking a breathable atmosphere but rich in minerals. While settlements on the moon have been destroyed to the extent that all that remains is debris scattered across the terrain, some automated facilities are still functioning and see a constant flow of cargo drones that stop by the moon to gather raw materials before quickly bringing them to the nearby station’s reserves.`

Reviewer: ```suggestion
	description `Alix is a small moon of the gas giant Pathera, lacking a breathable atmosphere but rich in minerals. While settlements on the moon have been destroyed to the extent that all that remains is debris scattered across the terrain, some automated facilities are still functioning and see a constant flow of cargo drones that stop by the moon to gather raw materials before quickly bringing them to the nearby station's reserves.`
```

> 	description `Alix is a small moon of the gas giant Pathera, lacking a breathable atmosphere but rich in minerals. While settlements on the moon have been destroyed to the extent that all that remains is debris scattered across the terrain, some automated facilities are still functioning and see a constant flow of cargo drones that stop by the moon to gather raw materials before quickly bringing them to the nearby station's reserves.`

Reviewer 2: ```suggestion
	description `Alix is a small moon of the gas giant Pathera, lacking a breathable atmosphere but rich in minerals. While settlements on the moon have been destroyed to the extent that all that remains are debris scattered across the terrain, some automated facilities are still functioning and see a constant flow of cargo drones that stop by the moon to gather raw materials before quickly bringing them to the nearby station's reserves.`
```

planet Ceilia'sei
	attributes ka'het uninhabited
	landscape land/station14
	description `Apart from one large, solid module, very little of this space station remains intact: the cloud of debris surrounding the core makes docking with it very difficult, and one side of the station even appears to have been blown up. The other part, however, has suffered much less damage, and the surface bristles with dozens of well-maintained turrets.`

planet Celeatis
	attributes ka'het uninhabited
	landscape land/canyon14

> 	description `Celeatis is a barren planet with mountains covering most of its surface.. Even in the few flat highlands, the terrain is too sterile for any kind of vegetation to grow naturally. On some slopes, the outline of an outpost can still be recognized, but they are all uninhabited: some of the villages have been partially destroyed, and large amounts of dust cover the remains to the point that the original color can't be recognized anymore.`

Reviewer 3: ```suggestion
	description `Celeatis is a barren planet with mountains covering most of its surface. Even in the few flat highlands, the terrain is too sterile for any kind of vegetation to grow naturally. On some slopes, the outline of an outpost can still be recognized, but they are all uninhabited: some of the villages have been partially destroyed, and large amounts of dust cover the remains to the point that the original color can't be recognized anymore.`
```

planet Osaeli
	attributes ka'het uninhabited
	landscape land/fields15

> 	description `An Earth-like planet with high biodiversity, at first glance Osaeli appears to be an idyllic world where no lifeform ever became the predominant one; but looking more closely, the signs of an ancient devastation can still be seen. In the middle of the forests, huge amounts of debris are scattered across large areas, and scans reveal many underground bunkers, partly destroyed and filled with water.`

Reviewer: ```suggestion
	description `An Earth-like planet with high biodiversity, at first glance Osaeli appears to be an idyllic world where no lifeform ever became the predominant one, but, looking more closely, the signs of an ancient devastation can still be seen. In the middle of the forests, huge amounts of debris are scattered across large areas, and scans reveal many underground bunkers, partly destroyed and filled with water.`
```

> 	description `An Earth-like planet with high biodiversity, at first glance Osaeli appears to be an idyllic world where no lifeform ever became the predominant one, but, looking more closely, the signs of an ancient devastation can still be seen. In the middle of the forests, huge amounts of debris are scattered across large areas, and scans reveal many underground bunkers, partly destroyed and filled with water.`

Reviewer 4: ```suggestion
	description `An Earth-like planet with high biodiversity, at first glance Osaeli appears to be an idyllic world where no lifeform ever became predominant; but looking more closely, the signs of an ancient devastation can still be seen. In the middle of the forests, huge amounts of debris are scattered across large areas, and scans reveal many underground bunkers, partly destroyed and filled with water.`
```

planet Palarei
	attributes ka'het uninhabited
	landscape land/valley5

> 	description `While its short distance from Gerenus makes it seem only barely capable of supporting life, Palarei’s surface is full of thickets of a small spiky bush native of this planet. Oddly enough, very few settlements on the planet appear to have been destroyed by weapon fire; most facilities are covered in meters of dust and sand, but other than that appear to be quite operative.`

Reviewer: ```suggestion
	description `While its short distance from Gerenus makes it seem only barely capable of supporting life, Palarei's surface is full of thickets of a small spiky bush native to this planet. Oddly enough, very few settlements on the planet appear to have been destroyed by weapon fire; most facilities are covered in meters of dust and sand but, other than that, appear to be quite operative.`
```

planet Retilie
	attributes ka'het uninhabited
	landscape land/sea19

> 	description `Retilie is an oceanic world; more than 95% of the planet's surface is covered by kilometers deep oceans. Few traces of coastal villages are still visible, mostly composed of ancient metallic formations standing some kilometers from the seaside, where the occasional high waves can’t reach them. A few small cities on the mainland are in a better state, but without anyone to maintain them, vegetation has grown to completely cover many of the surviving buildings.`

Reviewer 4: ```suggestion
	description `Retilie is an oceanic world; more than 95% of the planet's surface is covered by oceans kilometers deep. Few traces of coastal villages are still visible, mostly composed of ancient metallic formations standing some kilometers from the seaside, where the occasional high waves can’t reach them. A few small cities on the mainland are in a better state, but without anyone to maintain them, vegetation has grown to completely cover many of the surviving buildings.`
```

Reviewer 5: I agree with this change, why was it resolved?

Author: some issue

> 	description `Retilie is an oceanic world; more than 95% of the planet's surface is covered by oceans kilometers deep. Few traces of coastal villages are still visible, mostly composed of ancient metallic formations standing some kilometers from the seaside, where the occasional high waves can’t reach them. A few small cities on the mainland are in a better state, but without anyone to maintain them, vegetation has grown to completely cover many of the surviving buildings.`

Reviewer 4: I thought it was a contraction of "oceans that are kilometers deep", kilometers deep oceans did not sound as good to me.

> 	description `Retilie is an oceanic world; more than 95% of the planet's surface is covered by oceans kilometers deep. Few traces of coastal villages are still visible, mostly composed of ancient metallic formations standing some kilometers from the seaside, where the occasional high waves can’t reach them. A few small cities on the mainland are in a better state, but without anyone to maintain them, vegetation has grown to completely cover many of the surviving buildings.`

Reviewer 4: Oceans more than a kilometer deep are common on Earth, and I think this might imply that the oceans are only kilometers deep in some areas, instead of in most areas.

> 	description `Retilie is an oceanic world; more than 95% of the planet's surface is covered by oceans kilometers deep. Few traces of coastal villages are still visible, mostly composed of ancient metallic formations standing some kilometers from the seaside, where the occasional high waves can’t reach them. A few small cities on the mainland are in a better state, but without anyone to maintain them, vegetation has grown to completely cover many of the surviving buildings.`

Reviewer 6: ```suggestion
	description `Retilie is an oceanic world; more than 95% of the planet's surface is covered by oceans, which can be several kilometers deep. Few traces of coastal villages are still visible, mostly composed of ancient metallic formations standing some kilometers from the seaside, where the occasional high waves can’t reach them. A few small cities on the mainland are in a better state, but without anyone to maintain them, vegetation has grown to completely cover many of the surviving buildings.`
```

How's this for a slight variation? I think it implies a greater overall depth than the previous suggestion.

planet Siteria
	attributes ka'het uninhabited
	landscape land/water0

> 	description `An Earth-like planet, Siteria is notable for the large number of settlements on it: every few kilometers there are remains of sizable villages, some of which still haven’t been completely covered by vegetation even today. A few cities even have tall buildings still partially held together, a testament to how durable the constructions of the planet’s original inhabitants were.`

Reviewer: ```suggestion
	description `An Earth-like planet, Siteria is notable for the large number of settlements on it: every few kilometers there are remains of sizable villages, some of which still haven't been completely covered by vegetation even today. A few cities even have tall buildings still partially held together, a testament to how durable the constructions of the planet's original inhabitants were.`
```
